### PR TITLE
support es6 syntax & gulp serve

### PR DESCRIPTION
**Pull request by qcgm1978**

### CODE REVIEW

**Comment by qcgm1978** [comment]
babel config file added

File: gulpfile.babel.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
+'use strict';
+// Utilities
+import gulpLoadPlugins from 'gulp-load-plugins';
+const $ = gulpLoadPlugins();
+import browserSync from 'browser-sync';
+const reload = browserSync.reload;
+var autoprefixer = require('autoprefixer');
```
Comment (copycut) [inline gulpfile.babel.js]: Use const or import

File: gulpfile.babel.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
+'use strict';
```
Comment (copycut) [inline gulpfile.babel.js]: You can remove 'use strict' inserted by default by babel

**Comment by eltonmesquita** [comment]
Although I do like the idea a lot, I don't think it's needed at the moment. But I was seriously thinking about dropping gulp and use a simpler build system using only npm for the sass branch - version 4.x.
Would you be willing to help?

**Pull request closed without merging**